Feat/recaps (#221)
codatio/codat-docs: docs/using-the-api/managing-companies.md

updatedAt: "2022-12-21T06:24:11.564Z"
---

## Onboarding your users

> Your users/customers are [companies](/core-concepts/companies). To access their data you'll need to onboard them.

Reviewer: Suggestion:

Your users or customers are represented as companies.

> You'll have to go through the following steps:

Reviewer: ```suggestion
To onboard a new user or customer:
```

1. Create a company
2. Authorize access to sources of data
3. Pull the data

You can either onboard users:

- When they first create an account with your service

> - At the first point you wish to retrieve their financial data

Reviewer: ```suggestion
- At the first point you want to retrieve their financial data
```

The `id` property that you receive in the response is the unique Codat identifier for this company. We recommend that you retain it for future reference.
:::

### Authorize access

> Once you've created the company, they'll need to give you permission to pull their data. There are several approaches to this, but for simplicity we've just cover our out-of-the-box [hosted link](/auth-flow/authorize-hosted-link) approach.

Reviewer: ```suggestion
Once you've created the company, they'll need to give you permission to pull their data. There are several approaches to doing this, but for simplicity we've just covered our out-of-the-box [hosted link](/auth-flow/authorize-hosted-link) approach.
```

Get a `redirect` URL for the company by following the process [here](/auth-flow/authorize-hosted-link). Send the user to the `redirect` URL. They will be prompted to select their accounting software and complete the linking process using the [Link flow](/auth-flow/overview).

Once the user finishes the Link flow, they will be redirected back to the Redirect URL, as described [earlier in this guide](/using-the-api/managing-companies#redirect-the-user). At this point the re-authorization process is complete and their data has begun synchronizing again.

## Deleting companies

> You can delete companies in the Portal in the table in **Companes**.

Reviewer: ```suggestion
You can delete companies in the Portal in the table in **Companies**.
```
